index.html

```diff
@@ -108,6 +108,34 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+        <div class="card">
+          <p class="name">Emily Lubkert</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/emilylub" target="_blank">@emilylub</a>
+          </p>
+          <p class="about">I am an elementary educator turned full stack developer.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://missing.csail.mit.edu/2020/course-shell/" target="_blank" title="First Resource">
+                  The Missing Semester of Your CS Education
+                </a>
+              </li>
+              <li>
+                <a href="https://24ways.org/2012/how-to-make-your-site-look-half-decent/" target="_blank" title="Second Resource">
+                  How to Make Your Site Look Half-Decent in Half an Hour
+                </a>
+              </li>
+              <li>
+                <a href="https://github.com/lydiahallie/javascript-questions#readme" target="_blank" title="Third resource">
+                  JavaScript Questions
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
 
```

Comment:
Please keep the comment 
`<!-- ________*END TEMPLATE* Contributor card END ________ -->`  and change it to
to `<!-- ________ Your_Name Contributor card END ________ -->`